doc/tutorials/rtvm_workload_design_guideline.rst

```diff
@@ -0,0 +1,42 @@
+.. _rtvm_workload_guideline:
+
+Guidelines to the workload design in an RTVM
+############################################
+
+For a RTOS designer/developer, if he/she want to run the RTOS based on ARCN RTVM,
+he/she must be aware of the difference between native platform and virtual machine,
+especially those will impact real time capability.
+Generally speaking, the designer/developer should try to avoid VM-Exit when running
+a RT thread, or he/she must consider adding additional margin of CPU cycles for the
+cost of VM-Exit.
+
+Best Practice: Run RTVM with dedicated resources/devices
```

Comment:
Is this the only topic that's a "Best Practice"? I'd recommend leaving off the "Best Practice:" beginning of this title.